src/data/events/EventsD2ApiRepository.ts

```diff
@@ -81,34 +82,36 @@ export class EventsD2ApiRepository implements EventsRepository {
         const orgUnits = cleanOrgUnitPaths(orgUnitPaths);
 
         const fetchApi = async (orgUnit: string, page: number) => {
-            return this.api.events
+            return this.api.tracker.events
                 .get({
                     pageSize: 250,
                     totalPages: true,
                     page,
                     orgUnit,
-                    startDate: period !== "ALL" ? startDate.format("YYYY-MM-DD") : undefined,
-                    endDate: period !== "ALL" ? endDate.format("YYYY-MM-DD") : undefined,
-                    lastUpdated: lastUpdated ? moment(lastUpdated).format("YYYY-MM-DD") : undefined,
+                    occurredAfter: period !== "ALL" ? startDate.format("YYYY-MM-DD") : undefined,
+                    occurredBefore: period !== "ALL" ? endDate.format("YYYY-MM-DD") : undefined,
+                    updatedAfter: lastUpdated ? moment(lastUpdated).format("YYYY-MM-DD") : undefined,
                     fields: { $all: true },
                 })
                 .getData();
         };
 
-        const result = await promiseMap(orgUnits, async orgUnit => {
-            const { events, pager } = await fetchApi(orgUnit, 1);
+        const result: D2TrackerEvent[][] = await promiseMap(orgUnits, async orgUnit => {
```

Comment:
[minor (and even subjective)] a more orthodox approach is to move the type as a restriction of the returned block instead as the variable type (closer to where the error may pop up). That's it: 

```ts
const result = await promiseMap(orgUnits, async (orgUnit): Promise<D2TrackerEvent[]> => {
```